### PR TITLE
[main] Update libmspack to v0.10.1alpha

### PR DESCRIPTION
**Pull request by henryli001**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- Update libmspack to the latest version 0.10.1alpha

###### Change Log  

- Upgrade to version 0.10.1alpha
- Remove binaries under /usr/bin
- Add README, TODO, ChangeLog, AUTHORS to the main package

###### Does this affect the toolchain?  

NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Local Build

### CODE REVIEW

File: SPECS/libmspack/libmspack.spec [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+Release:        1%{?dist}
 License:        LGPLv2+
-URL:            http://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
-Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
 Source0:        http://www.cabextract.org.uk/libmspack/%{name}-%{version}.tar.gz
 %define sha1    libmspack=073348180586d7b0f61fd7f971162ffb5c1f6621
```
Comment (thcrain-msft) [inline SPECS/libmspack/libmspack.spec:11]: Please remove the sha1 line

Comment (henryli001) [reply SPECS/libmspack/libmspack.spec]: Fixed

File: SPECS/libmspack/libmspack.spec [hunk 2]
```diff
@@ -1,20 +1,21 @@
 Summary:        A library that provides compression and decompression of file formats used by Microsoft
 Name:           libmspack
-Version:        0.7.1alpha
-Release:        4%{?dist}
+Version:        0.10.1alpha
```
Comment (jslobodzian) [inline SPECS/libmspack/libmspack.spec:3]: Is there a non-alpha version available?

Comment (henryli001) [reply SPECS/libmspack/libmspack.spec:3]: No, all of its versions are tagged with alpha at the end

File: SPECS/libmspack/libmspack.spec [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+Release:        1%{?dist}
 License:        LGPLv2+
-URL:            http://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
-Group:          Applications/System
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/System
+URL:            https://www.cabextract.org.uk/libmspack/libmspack-0.5alpha.tar.gz
```
Comment (thcrain-msft) [inline SPECS/libmspack/libmspack.spec:9]: Can we change the URL to not reference the old tarball?

Comment (henryli001) [reply SPECS/libmspack/libmspack.spec]: Fixed

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**